warzone/Africa.map

```diff
@@ -0,0 +1,68 @@
+[Map]
+author=Sean O'Connor
+warn=yes
+image=Africa.bmp
```

Comment:
Bless the rains

Reply:
THERE'S NOTHING THAT A HUNDRED MEN OR MORE COULD EVER DOOOOOOOOOOO